src/components/Section/Package/Submit/__snapshots__/index.test.jsx.snap

```diff
@@ -91,7 +91,7 @@ Not a guarantee of acceptance, but all required fields are complete."
           onClick={[Function]}
           type="button"
         >
-          Submit your SF-86
+          Submit your
```

Comment:
@joshmfranklin Ideally this snapshot shouldn't change. This can be fixed by providing a `formType` prop to the test case. lmk if you want to look at it together!

Reply:
@suzubara yeah I'd love to look at that together!